src/mlpack/methods/ann/layer/max_pooling.hpp

```diff
@@ -56,7 +56,7 @@ class MaxPooling
   MaxPooling();
 
   /**
-   * Create the MaxPooling object using the specified number of units.
+   * Create the MaxPooling object using the specified parameter.
```

Comment:
```suggestion
   * Create the MaxPooling object using the specified parameters.
```
Not sure, maybe this is an improvement? :)